guest/testdata/cyclestate/main.go

```diff
@@ -176,6 +178,15 @@ func (statePlugin) NormalizeScore(state api.CycleState, pod proto.Pod, _ api.Nod
 	return
 }
 
+func (statePlugin) Reserve(state api.CycleState, pod proto.Pod, nodeName string) (status *api.Status) {
+	mustFilterState(state)
```

Comment:
nit:
I thought that this code wouldn't pass `Test_guestPool_bindingCycles` because this test doesn't call Filter function. It is very surprising for me
Just to clarify, is the testdata/cyclestate/main.wasm latest one?

Reply:
Yes, you are right. It passed because `Test_guestPool_bindingCycles` doesn't call `Reserve()`. I'll make it clarify.

Reply:
Ah I see, but Test_guestPool_bindingCycles also calls unreserve, right?